docs/source/_json/batching.json

```diff
@@ -0,0 +1,48 @@
+GET /plone/folder/@search?b_size=5&sort_on=path
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+{
+  "@id": "http://localhost:55001/plone/folder/@search?sort_on=path", 
```

Comment:
@tisto no, this is supposed to be the canonical ID to the base collection without any batching information (loosely based on https://www.w3.org/community/hydra/wiki/Collection_Design#Pagination). The ID for the current batch is in `batching/@id`.

I guess the `sort_on` parameter doesn't really affect the collection's contents, just their ordering. But if you consider other query string parameters for `@search` (or possibly other endpoints), the query string params can directly affect what resource will be returned, so I'm preserving them and just stripping batching related params for the canonical resource URL.